src/schema/index.ts

```diff
@@ -50,6 +50,36 @@ const Schema = gql`
       user: ID
     ): Profile
   }
-`
+  type ratingSystem {
+    name:String!
+    grade:[Int]!
+    description:[String]!
+    percentage:[String]!
+  }
+  type Mutation {
+    createRatingSystem(name: String!, grade:[Int]!, description: [String]!, percentage: [String]!): ratingSystem!
+  }
+  type Query{
+    getRatingSystem:[ratingSystem]
```

Comment:
Does this return one or many systems.

If it the first, you can change the query to `getRatingSystems`